### PR TITLE
OMPI: update code for getting version (ep. 2)

### PR DESCRIPTION
**Pull request by amaslenn**

### CODE REVIEW

**Comment by amaslenn** [comment]
@jsquyres @alinask @miked-mellanox please review

File: lib/MTT/Values/Functions/MPI/OMPI.pm [hunk 1]
```diff
@@ -167,19 +167,18 @@ sub get_version {
 
     my $match = qr/ompi:version:full:(.*)$/;
 
-    if (not open INFO, "$bindir/ompi_info --parsable|") {
-        open INFO, "$bindir/ompi_info -V|";
+    my @out = `$bindir/ompi_info --parsable`;
+    if ($? >> 8) {
+        @out = `$bindir/ompi_info -V`;
```
Comment (miked-mellanox) [inline lib/MTT/Values/Functions/MPI/OMPI.pm]: missing check if ompi_info succeeded

Comment (amaslenn) [reply lib/MTT/Values/Functions/MPI/OMPI.pm]: Don't need it. Next loop will match nothing and return it. Very backward compatible :)

Comment (miked-mellanox) [reply lib/MTT/Values/Functions/MPI/OMPI.pm]: but no next loop here? it is inside "if"
what do I miss

Comment (amaslenn) [reply lib/MTT/Values/Functions/MPI/OMPI.pm]: `for` for `@out` is outside this `if`, closing `}` is on line 174.

**Pull request merged**